proto/composable/stakingmiddleware/v1beta1/genesis.proto

```diff
@@ -11,4 +11,5 @@ option go_package = "x/stakingmiddleware/types";
 // GenesisState defines the stakingmiddleware module's genesis state.
 message GenesisState {
   Params params = 1 [ (gogoproto.nullable) = false ];
+  RewardDenom reward_denom = 2 [ (gogoproto.nullable) = false ];
```

Comment:
Does the new field in Genesis state does not require any upgrade handler?
Did you check that new binary works good on the prev storage for blockchain where this field does not exists?